src/containers/NFT/NFTHeader/test/Details.test.js

```diff
@@ -40,7 +42,17 @@ describe('NFT Details container', () => {
 
   it('renders defined fields', () => {
     const wrapper = createWrapper()
-    expect(wrapper.find('.row').length).toEqual(3)
+    expect(wrapper.find('.row').length).toEqual(4)
+    expect(
```

Comment:
If you do `expect(wrapper.text()).toEqual(expect.stringContaining('bafybeigdyrzt5sfp7udm7hu76uh7y26nf4dfuylqabf3oclgtqy55fbzdi'))` You get a cleaner message when it fails.  How it is now will fail with "Expected true to be false" or something like that.